packages/web3torrent/src/components/site-budget-table/SiteBudgetTable.tsx

```diff
@@ -1,79 +1,64 @@
+    key => channelState[key].beneficiary === mySigningAddress
   );
 
   const spent = myPayingChannelIds
-    .map(id => channelCache[id].beneficiaryBalance)
+    .map(id => channelState[id].beneficiaryBalance)
     .reduce((a, b) => bigNumberify(a).add(bigNumberify(b)), bigNumberify(0));
 
   const received = myReceivingChannelIds
-    .map(id => channelCache[id].beneficiaryBalance)
+    .map(id => channelState[id].beneficiaryBalance)
     .reduce((a, b) => bigNumberify(a).add(bigNumberify(b)), bigNumberify(0));
 
-  const spendBudget = bigNumberify(budgetCache.budgets[0].availableSendCapacity);
+  const spendBudget = bigNumberify(budget.budgets[0].availableSendCapacity);
+
+  const receiveBudget = bigNumberify(budget.budgets[0].availableReceiveCapacity);
 
-  const receiveBudget = bigNumberify(budgetCache.budgets[0].availableReceiveCapacity);
   return (
-    <Web3TorrentContext.Consumer>
-      {web3Torrent => {
-        return (
-          <>
-            <table className="site-budget-table">
-              <thead>
-                <tr className="budget-info">
-                  <td className="budget-button">Wallet Action</td>
-                  <td className="budget-number"> Spent / Budget </td>
-                  <td className="budget-number"> Earned / Budget </td>
-                </tr>
-              </thead>
-              <tbody>
-                <tr className="budget-info">
-                  <td className="budget-button">
-                    <button
-                      id="budget-withdraw"
-                      onClick={async () => {
-                        web3Torrent.paymentChannelClient.closeAndWithdraw();
-                      }}
-                    >
-                      Withdraw
-                    </button>
-                  </td>
-                  <td className="budget-number">
-                    {' '}
-                    {`${prettyPrintWei(spent)} / ${prettyPrintWei(spendBudget)}`}{' '}
-                  </td>
-                  <td className="budget-number">
-                    {' '}
-                    {`${prettyPrintWei(received)} / ${prettyPrintWei(receiveBudget)}`}{' '}
-                  </td>
-                </tr>
-              </tbody>
-            </table>
-          </>
-        );
-      }}
-    </Web3TorrentContext.Consumer>
+    <>
+      <table className="site-budget-table">
+        <thead>
+          <tr className="budget-info">
+            <td className="budget-button">Wallet Action</td>
+            <td className="budget-number"> Spent / Budget </td>
+            <td className="budget-number"> Earned / Budget </td>
+          </tr>
+        </thead>
+        <tbody>
+          <tr className="budget-info">
+            <td className="budget-button">
+              <button id="budget-withdraw" onClick={closeBudget}>
+                Withdraw
+              </button>
+            </td>
+            <td className="budget-number">
+              {' '}
```

Comment:
Are these needed? For me, sometimes, prettier (or maybe eslint?) insert these for reasons I don't understand.